app/services/users/delete_articles.rb

```diff
@@ -8,6 +8,7 @@ def call(user, cache_buster = CacheBuster)
       virtual_articles = user.articles.map { |article| Article.new(article.attributes) }
       user.articles.find_each do |article|
         article.reactions.delete_all
+        article.buffer_updates.delete_all
```

Comment:
@Zhao-Andy Please let me know if there's anything else we need to do in regards to this.

Reply:
That should be it! Thanks for adding this.